gravitee-apim-console-webui/src/services-ngx/environment-identity-provider.service.ts

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Inject, Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { Observable } from 'rxjs';
+
+import { Constants } from '../entities/Constants';
+import { IdentityProviderActivation } from '../entities/identity-provider';
+import { ActivatedIdentityProvider } from '../management/settings/identity-providers-ng/identity-providers-ng.component';
```

Comment:
🤔 service imports must not come from components but from entities

Reply:
maybe more IdentityProviderActivation ?

Reply:
Okay, currently I'm using `IdentityProviderActivation` which I set to have only one required parameter. `EnvironmentIdentityProviderService list()` is also using that interface and there was always one parameter.